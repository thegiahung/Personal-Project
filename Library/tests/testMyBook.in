add collection mybooks.csv
list genres
list authors
list all long
add member Python
add member Java
add member C
rent 100000 111112
rent 100000 111113
relinquish all 100000
rent 100001 111114
relinquish 100001 111114
exit