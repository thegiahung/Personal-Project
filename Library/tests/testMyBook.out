user: 17 books successfully added.

user:  Fantasy
Action
Comedy
Fantasy
Harem
Isekai
Musical
Romance
Science Fiction

user: Akasaka Aka
Fuji TV
Koshi Tachibana
Kugane Maruyama
Misora Riku
Miyajima Reiji
Miyazaki Yuu
Nagai Tatsuyuki
Reki Kawahara
Sato Tsutomu
Shimizu Yu
Shin-Ei Animation
Shirakome Ryo
Shu
Yoshiaki Fujisawa
Yusagi Aneko

user: 111111: Sword Art Online (Reki Kawahara,  Fantasy)
Currently available.

111112: Accel World (Reki Kawahara, Fantasy)
Currently available.

111113: Your Lie In April (Fuji TV, Musical)
Currently available.

111114: Date A Live (Koshi Tachibana, Romance)
Currently available.

111115: Takagi (Shin-Ei Animation, Romance)
Currently available.

111116: Kaguya-sama (Akasaka Aka, Romance)
Currently available.

111117: Kanojo Okarashimasu (Miyajima Reiji, Comedy)
Currently available.

111118: Gakuen Toshi Asterisk (Miyazaki Yuu, Science Fiction)
Currently available.

111119: Rakudai Kishi No Calvary (Misora Riku, Action)
Currently available.

111120: Ano Hi Mita Hana no Namae o Bokutachi wa Mada Shiranai (Nagai Tatsuyuki, Musical)
Currently available.

111121: The Misfit of Demon King Academy (Shu, Harem)
Currently available.

111122: Love Live! School Idol Project (Yoshiaki Fujisawa, Musical)
Currently available.

111123: Bladedance of Elementalers (Shimizu Yu, Fantasy)
Currently available.

111124: The Iruity at Magic High School (Sato Tsutomu, Action)
Currently available.

111125: Overlord (Kugane Maruyama, Isekai)
Currently available.

111126: The Rising of the Shield Hero (Yusagi Aneko, Isekai)
Currently available.

111127: Arifureta: From Commonplace to World's Strongest (Shirakome Ryo, Harem)
Currently available.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Ending Library process.
